src/components/addressform.main.jsx

```diff
@@ -426,10 +426,10 @@ class AddressFormMain extends React.Component {
           <div className="form-group">
             <div className="control-label" />
             <div className="form-input btn-container">
-              <button className="btn btn-primary address-save-btn" data-el-label="addressForm.save" type="submit">
+              <button className="vestri-btn primary address-save-btn" data-el-label="addressForm.save" type="submit">
```

Comment:
minor comment. Can we change the class name of the button from "vestri-btn" to "styled-btn" or something more generic? Just so when it comes to branding, the styles don't contain the "vestri" name.

Reply:
Renamed to `ep-btn`